e2e/cypress/integration/system_console/search_box_spec.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+import * as TIMEOUTS from '../../fixtures/timeouts';
```

Comment:
Please add an extra line after the group line before the import statement (as pointed earlier by Joseph)

Reply:
Done! Thanks for pointing it out. I've also removed the entire `prod` comment line. Hope that's what you wanted (or if you only wanted the `prod` portion removed and not the entire line). Thanks!